datafusion/core/src/datasource/statistics.rs

```diff
@@ -91,10 +91,10 @@ pub async fn get_statistics_with_limit(
                 // counts across all the files in question. If any file does not
                 // provide any information or provides an inexact value, we demote
                 // the statistic precision to inexact.
-                num_rows = add_row_stats(file_stats.num_rows.clone(), num_rows);
+                num_rows = add_row_stats(&file_stats.num_rows, &num_rows);
```

Comment:
Are we able to have `Precision` to be `Copy` when `T` is `Copy`? That would simplify things.

Reply:
THank you @Dandandan  -- that was a great idea -- I did it in in 7b71ea5af and it worked great

Reply:
Nice 👌